sdcm/cluster.py

```diff
@@ -250,6 +250,7 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
 
         self.remoter: Optional[RemoteCmdRunnerBase] = None
 
+        self._use_dns_names: bool = parent_cluster.params.get('use_dns_names') or False
```

Comment:
we usually set it in one of the config files, either specific to backend, like `aws_config.yaml` or `test_default.yaml`, so setting it in here like this, will be much harder to understand the defaults, when someone will decide to change it